stdlib/Include/liblocale/liblocale.c

```diff
@@ -0,0 +1 @@
+#include<locale.h>
```

Comment:
I am trying to understand how are you going to solve it.
If this file only includes `<locale.h>`, what does cmake do here?
what kind of symbols will `liblocale` obtains?

Reply:
If you only need the single function, you can manually define an extern function.
https://doc.rust-lang.org/std/keyword.extern.html

If you need many symbols, using [bindgen](https://github.com/rust-lang/rust-bindgen) is another way to generate it.